front/app/modules/commercial/insights/messages.ts

```diff
@@ -46,4 +33,42 @@ export default defineMessages({
     id: 'app.containers.Admin.Insights.discover',
     defaultMessage: 'Discover how it works',
   },
+  insightsTitle: {
+    id: 'app.containers.Admin.Insights.title',
+    defaultMessage: 'Insights',
+  },
+  insightsDescription: {
+    id: 'app.containers.Admin.Insights.description',
+    defaultMessage:
+      'Efficiently analyse the textual input on your platform. Group individual contributions that belong together. Understand what participants are really saying.',
+  },
+  insightsLink: {
+    id: 'app.containers.Admin.Insights.link',
+    defaultMessage: 'Read our support guide',
+  },
+  insightsListTitle: {
```

Comment:
We usually have the key matching the last part of the id, would be good to keep convention here :)